docs/cache/manage/limits.md

```diff
@@ -3,24 +3,85 @@ sidebar_position: 4
 |--------------------------------------------------------------------------------------------------------------------|----------------|
-| API rate per cache (data plane)                                                                                    | 100 requests/s |
-| API rate per customer (control plane)                                                                              | 5 requests/s   |
+| API rate per cache (data plane)                                                                                    | 100 operations/s |
+| API rate per customer (control plane)                                                                              | 5 operations/s   |
 | Throughput per cache                                                                                               | 1MB/s          |
 | Maximum item size                                                                                                  | 1MB            |
 | Max cache count (per account)                                                                                      | 10             |
 | Time to live (TTL)                                                                                                 | 1 day          |
 | Per [collection (CDT)](https://docs.momentohq.com/develop/datatypes#collection-data-types-cdts) element size limit | 128KB          |
-| Permissions per [API key or token](./../develop/api-reference/auth.md)  (hard limit)                    | 10             |
+| Permissions per [API key or token](./../develop/api-reference/auth.md)  (hard limit)                               | 10             |
 
 ## Soft limits and support
 
 The limits on this page are soft limits that can be altered unless specifically denoted. If you need limits adjusted, please reach out to [Momento Support](mailto:support@momentohq.com). Please include your login email, the name of the cache(s) to be altered, the cloud+region the cache is located in (e.g. AWS eu-west-1), and which limits from the list you’d like increased.
+
+## Operations
+
+Service limits are based on the number of operations performed per second. Some cache data plane APIs can perform multiple operations in a single request. Since batch operations are more efficient, the limit cost of these APIs is discounted at a 2:1 ratio. This means every two elements will count as one towards the limiter. For example, a `SetAddElements` request adding one or two elements costs one, but with three or four elements it costs two, etc.
+
+The below table describes how the number of operations is calculated for all cache APIs.
+
+| API                      | Operations                                                      |
+| ------------------------ | ------------                                                    |
+| Set                      | 1                                                               |
```

Comment:
Most of these are 1, so another way we could display this is only show the APIs that are not just 1 operation? But although that might look a little nicer, I think the explicitness is useful, so you can look for an API and know that it is actually 1 rather than that someone forgot to add an API to this list.